### ISSUE TITLE
Usecase Peminjaman Kunci Parkir

### ISSUE THREAD

**Issue by jeffrynasri** [aleknurf/FP-RK#3]
untuk usecase peminjaman itu perlu meminjam kunci parkir atau hanya Melihat Status Peminjaman Kunci Parkir?

karena di usecase hanya berutuliskan Melihat Status Peminjaman Kunci Parkir

**Comment by MHabiburRahman7** [comment]
Tidak,
Peminjaman itu hanya sebatas peminjaman kunci parkir saja

**Comment by jeffrynasri** [comment]
Berarti kelompok kami tidak menghandle peminjaman kunci ya? hanya sebatas melihat status peminjaman

**Comment by MHabiburRahman7** [comment]
Iya, asumsinya itu sudah ada halaman yang menunjukkan form peminjaman kunci. Tapi kelompok kalian tampilkan status kuncinya masih dipinjam atau sudah dikembalikan

**Comment by malanadya** [comment]
Form peminjaman kunci sudah dibuatkan oleh Lab AJK. SI Forkomlab hanya menampilkan informasi (berupa status peminjaman) yang didapat dari form tersebut, apakah kunci sedang dipinjam atau tidak. Semoga membantu :)
